api/api.go

```diff
@@ -2,26 +2,32 @@ package api
 
 import (
 	"context"
-
-	store "github.com/ONSdigital/dp-search-reindex-api/store"
+	"github.com/ONSdigital/dp-search-reindex-api/mongo"
+	"github.com/ONSdigital/log.go/log"
 	"github.com/gorilla/mux"
 )
 
 //JobStoreAPI provides a struct to wrap the api around
 type JobStoreAPI struct {
-	Router   *mux.Router
-	jobStore store.JobStore
+	Router  *mux.Router
+	mongoDB mongo.MgoJobStore
```

Comment:
I'd have kept this is a JobStore, the api package doesn't really care that it's storing the jobs in mongo, it only cares that the jobs are stored and can be retrieved.

Reply:
done